Create-Vms.ps1

```diff
@@ -12,13 +12,28 @@ param
 
     [ValidateSet("Delete","Leave","Error")]
     [Parameter(Mandatory=$true, HelpMessage="What to do if a VM with the same name exist in the lab (Delete, Leave, Error)")]
-    [string] $IfExist
+    [string] $IfExist,
+
+    [Parameter(Mandatory=$false, HelpMessage="Once the VMs are created, apply Windows Updates?")]
+    [bool] $ApplyWindowsUpdates = $false
```

Comment:
New feature - at the end of creating all the VMs, can apply windows updates so the VMs are in a current state out of the gate.